TESTING_GUIDE.md

```diff
@@ -29,7 +29,7 @@ We have devised our own test framework to test the spec against an OpenSearch cl
 
 ### Running Spec Tests Locally
 
-Downloading and installing latest version of Node.js and npm
+Download and install latest version of Node.js and npm from [here]
 
 https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
```

Comment:
To make "here" a link the syntax is "[here](link)", so:

```suggestion
Download and install the latest version of Node.js and npm from [here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
```

Reply:
Hit preview, this renders the following:

<img width="1068" alt="Screenshot 2024-08-21 at 10 02 18 AM" src="https://github.com/user-attachments/assets/f4206b6f-7600-4657-ab58-f2e6dc6cb8d6">

"here" needs to be a working link.